pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -394,6 +406,22 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {"class": "reduce", "mode": self.mode, "axes": self.axes, "from": self._get_input_layer_name(input)}
 
 
+class Length(Module):
+  """
+  Wraps RETURNN LengthLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, axis: Union[str, Dim]):
```

Comment:
Why `str`?

But maybe we want to allow `Dim` here (a dim tag). Although this mixes PyTorch and RETURNN things but passing `Dim` would only be used internally.

Reply:
`int` is missing, if this is ever supposed to be used for normal PyTorch code.

Reply:
I used `str` and `Dim` as that's what is allowed in RETURNN. Added `int` now.